edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_variables.scss

```diff
@@ -7,7 +7,7 @@ $secondary: #E89D00;
 $footer-background: #242E33;
 $footer-contrast: #F2F2F2;
 $dark: #313131;
-$gray: #E6E6E6;
+$gray: #242E33;
 
```

Comment:
```suggestion
$primary-light: #EE3F40;

```

Reply:
Would you consider add this variable and use it for the color border on line 59 of _navigation.scss